hack/setup-dev.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+root=$(dirname "${BASH_SOURCE[0]}")
+
+bash -e $root/install-dev-deps.sh
+source $root/dev-functions.sh
+
+# Create the control cluster
+kind delete cluster --name argocd-control-plane
+kind create cluster --name argocd-control-plane
+kubectl create namespace argocd
+
+# Setup argocd
+kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj-labs/applicationset/v0.1.0/manifests/install.yaml
```

Comment:
let's make the appset version an environment variable?

Reply:
Yeah, we should probably do the same for argocd as well. As what is stable can easily change underneath us.